scripts/zones/Ship_bound_for_Mhaura/Zone.lua

```diff
@@ -4,28 +4,95 @@
 --
 -----------------------------------
 local ID = require("scripts/zones/Ship_bound_for_Mhaura/IDs")
+require("scripts/globals/sea_creatures")
 -----------------------------------
 local zone_object = {}
 
 zone_object.onInitialize = function(zone)
 end
 
-zone_object.onZoneIn = function(player, prevZone)
+zone_object.onZoneIn = function(player, prevZone, zone)
     local cs = -1
-    if (player:getXPos() == 0 and player:getYPos() == 0 and player:getZPos() == 0) then
+    local zoneID = 221
+
+    if
+        player:getXPos() == 0 and
+        player:getYPos() == 0 and
+        player:getZPos() == 0
+    then
         local position = math.random(-2, 2) + 0.150
         player:setPos(position, -2.100, 3.250, 64)
+        if
+            player:getGMLevel() == 0 and
```

Comment:
the event doesn't activate. (This is to stop GMs from using !zone to the zones and breaking the event) 

We don't have anything currently that will let us auto start a transportation event. So this was my: Lets start the pirate event because a normal player is now in the zone.

Reply:
Is this acceptable for the time being then?

Reply:
My comment is unrelated to any mergeability decision, and I haven't personally reviewed, just read comments and decided to give everyone a heads up :+1: